core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -10,6 +10,9 @@ endif
 ifeq ($(PLATFORM_FLAVOR),juno)
 include core/arch/arm/cpu/cortex-armv8-0.mk
 platform-debugger-arm := 1
+# Workaround 808870: Unconditional VLDM instructions might cause an
+# alignment fault even though the address is aligned
+$(call force,CFG_TA_ARM32_NO_HARD_FLOAT_SUPPORT,y)
```

Comment:
But that will prevent the TA from using Advanced SIMD (NEON) altogether, right? Is it acceptable? It may be considered as a regression...

Reply:
I only prevents the compiler from generating the instructions. If you have assembly code it will still work, provided that you can avoid the problem described in the errata.

The errata is only applicable on the very first version (r0p0) of Cortex-A57 MPCore, so I suspect that it's not that widely used that it matters much.